main/krb5/APKBUILD

```diff
@@ -19,14 +13,23 @@ makedepends="$depends_dev libverto-dev openldap-dev openssl-dev
 options="suid !check"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-server
 	$pkgname-server-ldap:ldap $pkgname-pkinit $pkgname-libs"
-source="https://web.mit.edu/kerberos/dist/krb5/${_ver}/krb5-$pkgver.tar.gz
+_ver_maj_min() {
+	local v
+	for v; do
+		while true; do
+			case "${v}" in (*.*.*) v="${v%.*}" ;; (*.*) break ;; esac
+		done
+		printf -- '%s\n' "${v}"
+	done
+}
+_v="$(_ver_maj_min "${pkgver}")"
```

Comment:
I don't want to have forks in global scope.

Reply:
I reworked this to not use a sub-shell.